Cod3rsGrowth.Web/app.config

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
esse aquivo nao deve subir no git, so deve existir na sua máquina porque tem dados sensíveis, por exemplo, a senha e nome de usuário, quem for utilizar esse projeto tem que criar esse arquivo, deleta e coloca no git ignore